lib/player_manager.rb

```diff
@@ -1,23 +1,31 @@
-require 'game_state'
+require 'ai_basic'
+require 'ai_minimax'
+require 'game_rules'
 class PlayerManager
 
   attr_reader :current_player
 
-  def initialize(game_io, player1, player2)
-      @player1 = player1
-      @player2 = player2
+  Player = Struct.new(:mark, :type)
+
+  def initialize()
+      @player1 = Player.new(:X, :Human)
```

Comment:
On an unrelated note - symbols typically won't begin with an upper case.  snake_case is what you'll usually see.